### DISCUSSION TITLE
Convoluted cloudflare tunnel instructions (No insult intended just curious).

### DISCUSSION THREAD

**Discussion by blackstormlab** [dani-garcia/vaultwarden#4660] [Q&A]
### Discussed in https://github.com/dani-garcia/vaultwarden/discussions/4620

<div type='discussions-op-text'>

<sup>Originally posted by **StinkyTACO** June  5, 2024</sup>
is there any reason to follow the convoluted setup for a cloudflare tunnel with making the .yml and .json files and base64 decoding it instead of just using an environment variable with the tunnel token instead ?

somethig like this instead:

   cloudflare-tunnel:
    image: cloudflare/cloudflared:latest
    restart: always
    environment:
      - TUNNEL_TOKEN=tokenhere
    command: tunnel --no-autoupdate run</div>

**Comment by dfunkt** [comment]
The config above is pretty close to what I use, which implies configuring the tunnel via the web. The guide in the wiki is intended for a manual configuration of the tunnel.

**Comment by blackstormlab** [comment]
cool i wanted to make sure there was no reason to it because using it the other way just seems complicated for no real benefit.
